libnd4j/blas/cuda/NativeOps.cu

```diff
@@ -4083,6 +4083,22 @@ void NativeOps::initializeDevicesAndFunctions() {
     //cudaFuncGetAttributes(&funcAttributes[47], scalarAlongDimension_0_float);
     //cudaFuncGetAttributes(&funcAttributes[48], scalarAlongDimension_0_float16);
     //cudaFuncGetAttributes(&funcAttributes[48], scalarAlongDimension_0_double);
+
+    cudaFuncGetAttributes(&funcAttributes[49], concatKernelScalarInt);
+
+    cudaFuncGetAttributes(&funcAttributes[50], concatKernelScalarLong);
+
+    cudaFuncGetAttributes(&funcAttributes[51], concatKernelHStackInt);
+
+    cudaFuncGetAttributes(&funcAttributes[52], concatKernelHStackLong);
+
+    cudaFuncGetAttributes(&funcAttributes[53], concatKernelVStackInt);
+
+    cudaFuncGetAttributes(&funcAttributes[54], concatKernelVStackLong);
+
+    cudaFuncGetAttributes(&funcAttributes[55], concatKernelInt);
+
+    cudaFuncGetAttributes(&funcAttributes[56], concatKernelLong);
```

Comment:
get rid of this block please

this stuff is deprecated and will be removed before 1.0 release

Reply:
I will try to get rid, but I need to know what to the use instead where we currently have something like this: 
```
smem = funcAttributes[49].sharedSizeBytes;
concatKernelScalarInt<<< 128, 128, smem, *stream>>> 
```